tests/settings.py

```diff
@@ -29,6 +29,7 @@
         "aldryn_forms": None,
     },
     "DEFAULT_AUTO_FIELD": "django.db.models.AutoField",
+    "CMS_CONFIRM_VERSION4": True,
```

Comment:
This is required because the ci is pointing at develop-4, the community / DCA are activating fixing the shortfalls in V4 for community projects, issues such as reference to the old models "title" in the code (tech debt), the page tree (without djangocms-page-admin / djangocms-navigation installed), any other missing features deemed necessary for the community. I have no authority to stop any breaking changes coming through the community channels anymore. 

It has been agreed that v4 will have it's first community release on v4.1 which is being created in develop-4. 

Due to the above you are going to constantly hit changed features, I would advise to change the target branch of django-cms to one of the designated release branches such as release/4.0.1.x, if not you will be constantly hitting changes. 

If you're happy with hitting changes stay on develop-4.

Reply:
Have updated to the release branch to prevent this issue in the future.